website/docs/integrations/monday.mdx

```diff
@@ -4,39 +4,69 @@ title: monday.com - Manage feature flags from monday
 description: ConfigCat monday app. This is a step-by-step guide on how to connect and manage feature flags from monday.com boards and items.
 ---
 
-Turn features On / Off right from a linked item on your monday.com board with the <a href="https://monday.com/marketplace/10000079" target="_blank">ConfigCat Feature Flags</a> monday.com app. Also you can add targeting or percentage rules just as easily.
+The <a href="https://monday.com/marketplace/10000079" target="_blank">ConfigCat Feature Flags monday app</a> allows you to connect your Monday items and feature flags. Create or link existing flags to your items without leaving your monday instance.
 
-## Installation
+Turn features On/Off right from a linked item on your Monday board. You can also easily modify the linked flags to edit or add new Targeting or Percentage Rules.
 
-<img src="/docs/assets/monday/monday_auth.gif" className="zoomable" alt="Installation of the ConfigCat Feature Flags monday app" />
+This guide will help you with the plugin installation and familiarise you with the plugin usage.
+
+## Installation
 
 1. Install <a href="https://monday.com/marketplace/10000079" target="_blank">ConfigCat Feature Flags</a> monday.com app to your monday account.
-2. Open one of your boards in monday and click on an item.
-3. Add the ConfigCat Feature Flags monday app to your item.
-4. In order to use ConfigCat Feature Flags you have to authorize first with your ConfigCat Public API credentials.
-5. Get your ConfigCat Public API credentials: https://app.configcat.com/my-account/public-api-credentials
+2. Open one of your boards on monday and click on an Item.
```

Comment:
Ez biztosan "on" monday, és nem "in" monday?
Mert szerintem in "in" Jira, és biztosan nem "on" Jira.

Az "on monday" az szerintem azt jelenti, hogy "hétfőn".

Ennek ellenére, ha a Monday-esek "on monday"-ként szoktak beszélni a saját tooljukról, akkor felőlem maradhat így. Különben szerintem érdemes "in"-re cserélni.

Reply:
Esetleg lehetne on monday.com